.github/workflows/python-package.yml

```diff
@@ -22,6 +22,10 @@ jobs:
           cp docs/mkdocs.yml ./
       - name: Render
         run: uv run mkdocs build
+      - uses: actions/upload-artifact@v4
```

Comment:
Not mandatory, but since this is an awesome feature, you can add it to the `CONTRIBUTING.md` :) so contributors don't have to build the docs themselves. Just say 
- where to click to get the files
- unzip it
- open index.html with the browser

Again it's optional since you contributed this feature and it's a net improvement, but if you want to go the extra mile, I'd appreciate a note in the contributing guide.

Reply:
No problem, [added](https://github.com/gabrieldemarmiesse/python-on-whales/pull/656/files#diff-eca12c0a30e25b4b46522ebf89465a03ba72a03f540796c979137931d8f92055R43)!